modules/odc_con_customizing-a-developer-catalog-or-its-sub-catalogs.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: CONCEPT
+
+[id="odc_con_customizing-a-developer-catalog-or-its-sub-catalogs_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs
+
+The Red Hat OpenShift Container Platform Web Console provides support for the Administrator and the Developer perspectives. Cluster administrators have the ability to organize and manage a Developer catalog or its sub-catalogs based on the access control and privileges provided to a user. In some situations, customers do not want their users to have access to a particular Developer catalog or it's sub-catalogs. Customizing a user's role ensures that the user only views the Developer catalog or sub-catalogs resources assigned to them.
```

Comment:
```suggestion
Cluster administrators have the ability to organize and manage a Developer catalog or its sub-catalogs by enabling/disabling sub-catalog types or disabling the entire developer catalog. In some situations, customers do not want their users to have access to the Developer catalog or one or more of its sub-catalogs.
```

The user's role is not considered to Enable/Disable the developer catalog or its sub-catalog types.

Reply:
Fixed.